### ISSUE TITLE
reading netcdf files not written with veranda

### ISSUE THREAD

**Issue by sebhahn** [TUW-GEO/veranda#13]
[this assumption](https://github.com/TUW-GEO/veranda/blob/master/src/veranda/raster/native/netcdf.py#L162) cannot be made in general for netcdf files not written with veranda

**Comment by cnavacch** [comment]
Fair point, this was a simply first shot to separate the spatial dimensions from all other dimensions. In the old version of `veranda`  it was simply hard-coded `('x', 'y', 'time')`. 

The question is how we define it here. Either it is possible to retrieve this information from the file itself or the user has the possibility to specify it as an argument, e.g. with `space_dims` and `stack_dims`.
